### PR TITLE
Xtext Examples: Refactoring the test cases.

### PR DESCRIPTION
**Pull request by miklossy**
- Provide more compact and readable test code by utilizing Xtend
capabilities.

Signed-off-by: Tamas Miklossy <miklossy@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
I am not a fan of implicit return types. @kthoms @ArneDeutsch what do you think?

File: org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui.tests/src/org/eclipse/xtext/example/domainmodel/ui/tests/OutlineTest.xtend [hunk 1]
```diff
@@ -20,7 +20,7 @@ import org.junit.runner.RunWith
 @InjectWith(DomainmodelUiInjectorProvider)
 class OutlineTest extends AbstractOutlineTest {
 
-	@Test def void testOutline() {
+	@Test def testOutline() {
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui.tests/src/org/eclipse/xtext/example/domainmodel/ui/tests/OutlineTest.xtend]: please leave void

File: org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui.tests/src/org/eclipse/xtext/example/domainmodel/ui/tests/OutlineTest.xtend [hunk 2]
```diff
@@ -37,7 +37,7 @@ class OutlineTest extends AbstractOutlineTest {
 		)
 	}
 
-	@Test def void testOutlineWithPackage() {
+	@Test def testOutlineWithPackage() {
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui.tests/src/org/eclipse/xtext/example/domainmodel/ui/tests/OutlineTest.xtend]: please leave void

File: org.eclipse.xtext.xtext.ui.examples/projects/homeautomation/org.eclipse.xtext.example.homeautomation.tests/src/org/eclipse/xtext/example/homeautomation/tests/AbstractTest.xtend [hunk 3]
```diff
@@ -28,7 +28,7 @@ abstract class AbstractTest {
 		'''.test
 	}
 
-	@Test def void test2() {
+	@Test def test2() {
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui.examples/projects/homeautomation/org.eclipse.xtext.example.homeautomation.tests/src/org/eclipse/xtext/example/homeautomation/tests/AbstractTest.xtend]: please leave void

File: org.eclipse.xtext.xtext.ui.examples/projects/homeautomation/org.eclipse.xtext.example.homeautomation.tests/src/org/eclipse/xtext/example/homeautomation/tests/AbstractTest.xtend [hunk 4]
```diff
@@ -40,7 +40,7 @@ abstract class AbstractTest {
 		'''.test
 	}
 
-	@Test def void test3() {
+	@Test def test3() {
```
Comment (kthoms) [inline org.eclipse.xtext.xtext.ui.examples/projects/homeautomation/org.eclipse.xtext.example.homeautomation.tests/src/org/eclipse/xtext/example/homeautomation/tests/AbstractTest.xtend]: please leave void

Comment (kthoms) [reply org.eclipse.xtext.xtext.ui.examples/projects/homeautomation/org.eclipse.xtext.example.homeautomation.tests/src/org/eclipse/xtext/example/homeautomation/tests/AbstractTest.xtend]: also in other test cases

**Comment by miklossy** [comment]
Thanks for the feedback @cdietrich and @kthoms ! I updated the PR as requested.

**Pull request merged**